lib/src/main/java/vmath/Mat2.java

```diff
@@ -4,12 +4,26 @@ public class Mat2 {
     private final float[] m;
 
     public Mat2() {
-        m = new float[] {
-            1.0f, 0.0f,
-            0.0f, 1.0f
+        m = new float[]{
+                1.0f, 0.0f,
+                0.0f, 1.0f
         };
     }
 
+    public Mat2(float[] initialValues) {
```

Comment:
In other approved PRs the parameter was called something else (I think it was `m`).